csrc/ops/composite.h

```diff
@@ -81,6 +71,6 @@ TensorView* leaky_relu(TensorView* x, Val* negative_slope);
 
 NVF_API TensorView* view_as_real(TensorView* x);
 
-TensorView* eagerMatmul(TensorView* tv_a, TensorView* tv_b);
+TensorView* matmul(TensorView* tv_a, TensorView* tv_b);
```

Comment:
I suggest keeping the documentation in place.
```suggestion
// Matmul function which takes in tensors with the shapes
// A[M,K] B[K,N], but the tensors may have different layouts
// via strides. All restrictions from the matmul APIs also
// apply here.
TensorView* matmul(TensorView* tv_a, TensorView* tv_b);
```

Reply:
Added a comment detailing the functionality. The above comment is incorrect since we support all sizes now.